src/desktop/apps/isomorphic-relay-example/server.js

```diff
@@ -4,6 +4,7 @@ import { buildServerApp } from "reaction/Artsy/Router/server"
 import { routes } from "./routes"
 import { renderLayout } from "@artsy/stitch"
 import { Meta } from "./components/Meta"
+import { buildServerAppContext } from "desktop/lib/build_server_app_context"
```

Comment:
I would go ahead and ditch the snake case -- most new files are idiomatic camelCase.